browser/resources/settings/brave_wallet_page/brave_wallet_browser_proxy.ts

```diff
@@ -37,18 +37,21 @@ export type Option = {
 }
 
 export type SolanaProvider = Option
+export type CardanoProvider = Option
 
 export interface BraveWalletBrowserProxy {
   resetWallet: () => void
   resetZCashSyncState: () => void
   getWeb3ProviderList: () => Promise<string>
   getSolanaProviderOptions: () => Promise<SolanaProvider[]>
+  getCardanoProviderOptions: () => Promise<CardanoProvider[]>
   getTransactionSimulationOptInStatusOptions: () => Promise<Option[]>
   isNativeWalletEnabled: () => Promise<boolean>
   isBitcoinEnabled: () => Promise<boolean>
   isZCashEnabled: () => Promise<boolean>
   isZCashShieldedTxEnabled: () => Promise<boolean>
   isCardanoEnabled: () => Promise<boolean>
+  isCardanoDAppSupportEnabled: () => Promise<boolean>
   getAutoLockMinutes: () => Promise<number>
   getNetworksList: (coin: number) => Promise<NetworksList>
   getPrepopulatedNetworksList: () => Promise<NetworkInfo[]>
```

Comment:
Most values available through all these getters are runtime constants and should become a part of `loadTimeData` actually. But not now ofc